#!/usr/bin/env python
PACKAGE = "tcc"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()



gen.add("CtrlOmega_xy", double_t, 0, "A double parameter", 1.0, 0.1, 3.0)
gen.add("CtrlOmega_z", double_t, 0, "A double parameter", 2.2, 0.1, 3.0)

gen.add("PosErrorAccumulatedLimit_xyz", double_t, 0, "A double parameter",10, 1.0, 30)
gen.add("Pos_ki_xy", double_t, 0, "A double parameter", 0.02, 0, 1.0)
gen.add("Pos_ki_z", double_t, 0, "A double parameter", 0.2, 0, 1.0)


gen.add("k_p_yaw_", double_t, 0, "A double parameter", 0.812, 0.1, 3.0)

gen.add("k_I_yaw_", double_t, 0, "A double parameter", 0.02, 0, 0.5)

gen.add("yawErrorAccumLim_", double_t, 0, "A double parameterr", 1.57, 0, 3.14)

exit(gen.generate(PACKAGE, "tcc", "Param"))
